### PR TITLE
TLF-108 Initial questions about tenant 

### PR DESCRIPTION
**Pull request by MikeMaratheftis**
## What? 
Create 'Tenant details page & create Tenant's current address page with validations.
## Why? 
Pages have currently not been made.
## How? 
Created input fields with required validations.
## Testing? 
local testing only
## Check list

- [] I have reviewed my own pull request
- [ ] I have written tests (if relevant)

### CODE REVIEW

File: apps/lcs/translations/src/en/pages.json [hunk 1]
```diff
@@ -6,10 +6,22 @@
...
+    "legend": "Full name"
+  },
+  "tenant-dob": {
+    "header": "Date of birth"
+  },
+  "tenant-nationality": {
+    "header": "Country of nationality"
+  },
+  "ho-ref-number": {
+    "header": "Home Office reference number"
```
Comment (dk4g) [inline apps/lcs/translations/src/en/pages.json]: field related translations

File: apps/lcs/views/tenant-details.html [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    <div class="govuk-details__text">
+      <p1>You can provide any of these numbers:</p1>
+      <ul>
+        <li>biometric residence permit number</li>
+          <li>passport number (current or expired)</li>
+          <li>Home Office application reference number</li>
+          <li>any tracking reference number</li>
+      </ul>
+    </div>
+  </details>
```
Comment (dk4g) [inline apps/lcs/views/tenant-details.html]: would be good to extract it into content as it's language specific.

File: apps/lcs/views/content/what-counts-as-a-home-office-reference-number.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
+  <details class="govuk-details">
+    <summary class="govuk-details__summary">
+      <span class="govuk-details__summary-text">
+        What counts as a Home Office reference number
+      </span>
+    </summary>
+    <div class="govuk-details__text">
+      <p1>You can provide any of these numbers:</p1>
```
Comment (dk4g) [inline apps/lcs/views/content/what-counts-as-a-home-office-reference-number.md]: wrong tag

File: package.json [hunk 4]
```diff
@@ -25,7 +25,8 @@
   },
   "homepage": "https://github.com/UKHomeOffice/landlords-checking-service#readme",
   "dependencies": {
-    "hof": "^20.5.2"
+    "hof": "^20.5.2",
+    "accessible-autocomplete": "^2.0.4"
```
Comment (dk4g) [inline package.json]: could it be upgraded to "^3.0.0"?

File: assets/scss/app.scss [hunk 5]
```diff
@@ -26,5 +27,239 @@ p a[href] {
...
+      .list-entry {
+        width: 80%;
+        float: left;
+        margin-right: 20px;
+      }
+  
+      .list-entry-button {
+        margin: 0;
+      }
+    }
```
Comment (dk4g) [inline assets/scss/app.scss]: Seems this is not in use?

File: apps/lcs/views/content/what-counts-as-a-home-office-reference-number.md (hunk 3 above)
Comment (MikeMaratheftis) [reply apps/lcs/views/content/what-counts-as-a-home-office-reference-number.md]: Fixed

File: package.json (hunk 4 above)
Comment (MikeMaratheftis) [reply package.json]: Fixed

File: assets/scss/app.scss (hunk 5 above)
Comment (MikeMaratheftis) [reply assets/scss/app.scss]: Removed as not being used

**Review by dk4g — APPROVED** [review]
Dynamic placeholders in validation error messages will be covered separately 
[TLF-259](https://collaboration.homeoffice.gov.uk/jira/browse/TLF-259)

**Pull request merged**